spec/services/questions_service_spec.rb

```diff
@@ -121,6 +121,22 @@
       end
     end
 
+    context "when there is a Child Section 8 as Applicant" do
```

Comment:
I don't really understand what this scenario means 🤔 how can a 'Child Section 8' be an applicant?

Reply:
would
```
when there is a Child Section 8 proceeding with the client involvement type is Applicant
```
make more sense?